cms/src/api/home/content-types/home/schema.json

```diff
@@ -1,30 +1,25 @@
 {
-  "kind": "singleType",
-  "collectionName": "homes",
-  "info": {
-    "singularName": "home",
-    "pluralName": "homes",
-    "displayName": "home",
-    "description": ""
-  },
-  "options": {
-    "draftAndPublish": true
-  },
-  "pluginOptions": {},
-  "attributes": {
-    "Hero": {
-      "type": "component",
-      "repeatable": false,
-      "component": "sections.hero-section",
-      "required": true
-    },
-    "body": {
-      "type": "dynamiczone",
-      "components": [
-        "sections.news-section",
-        "general.button",
-        "general.card"
-      ]
-    }
-  }
+	"kind": "singleType",
+	"info": {
+		"singularName": "home",
+		"pluralName": "homes",
```

Comment:
I'd not remove the `collectionName` but rather change it to home. Also I'd rename the `pluralName` to home

Reply:
Results in an error:

2024-05-02 15:17:07 Error: The plural name "home" should be unique

Reply:
rip okay^^